numpy/linalg/linalg.py

```diff
@@ -531,6 +537,37 @@ def inv(a):
            [[-1.25,  0.75],
             [ 0.75, -0.25]]])
 
+    If a matrix is close to singular, the computed inverse may not satisfy
+    ``dot(a, ainv) = dot(ainv, a) = eye(a.shape[0])`` even if a `LinAlgError`
+    is not raised:
+
+    >>> a = np.array([[2,4,6],[2,0,2],[6,8,14]])
+    >>> inv(a)  # No errors raised
+    array([[-1.12589991e+15, -5.62949953e+14,  5.62949953e+14],
+       [-1.12589991e+15, -5.62949953e+14,  5.62949953e+14],
+       [ 1.12589991e+15,  5.62949953e+14, -5.62949953e+14]])
+    >>> np.dot(a, inv(a))
+    array([[ 0.  ,  0.  ,  0.  ],
+           [-0.5 ,  0.75,  0.25],
+           [ 0.  ,  0.  ,  1.  ]])
+
+    To detect ill-conditioned matrices, you can use `numpy.linalg.cond` to
+    compute its `condition number`. The larger the condition number, the more
```

Comment:
I would recommend a `Reference` to the [Wikipedia article on condition numbers](https://en.wikipedia.org/wiki/Condition_number). It provides some more context and a handy rule of thumb about interpreting the size of the condition number. We may also want to state that rule of thumb here.